verification_token/migrations/0004_migration.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.15 on 2020-11-12 13:38
+
+from django.db import migrations, models
+from django.utils.timezone import timedelta
+
+
+def recompute_value_for_new_field(apps, schema_editor):
+    VerificationToken = apps.get_model('verification_token', 'VerificationToken')
+    expiring_tokens_qs = VerificationToken.objects.filter(expiration_in_minutes__isnull=False)
+    expiration_minutes = set(expiring_tokens_qs.values_list('expiration_in_minutes', flat=True))
+
+    for minute_number in expiration_minutes:
+        expiring_tokens_qs.filter(expiration_in_minutes=minute_number).update(
```

Comment:
I am quite sure this style of migration will stir up controversy, but it's the only reasonable way I could find that does not require deleting old tokens (after all, this is a public library and we should not make unexpected changes for downstream) and that does not require to go through all the tokens one-by-one and changing them as Python objects (which would be veeery expensive with large number of tokens).

Reply:
Yes, there is no need to delete old tokens. This migration is actually quite clever :+1: